slider/__main__.py

```diff
@@ -1,34 +1,34 @@
-import click
-
 from . import Library
 
+if __name__ == '__main__':
+    try:
+        import click
+    except ImportError:
+        raise ImportError("click must be installed to use the slider cli")
```

Comment:
Like below, let's `raise from` the original error, but I like the additional context